appdata.patch

```diff
@@ -5,7 +5,7 @@
      <binary>minuet</binary>
    </provides>
 +  <releases>
-+    <release version="20.04.3" date="2020-07-07"/>
++    <release version="0.4.0" date="2017-04-12"/>
```

Comment:
I put `date="2017-04-12"` is from https://invent.kde.org/education/minuet/-/commit/db5f5c1ed89d74027742529f76a635ea882259f2#9a2aa4db38d3115ed60da621e012c0efc0172aae_13_12